todo_app/data/trello_items.py

```diff
@@ -29,19 +26,19 @@ def get_items(status):
                                                                   
     return list_of_cards
 
-def get_open_items():
+def get_do_items():
```

Comment:
The main idea was to have the logic to split the items into Do/Doing/Done in the ViewModel, not here, but this works as well

Reply:
Thanks. Jack has had me reset my code as he could foresee problems with modules further down the line so this code is no longer applicable btw.